### PR TITLE
Add eslint w/ airbnb rules and fix linting errors

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true
+    },
+    "extends": [
+        "airbnb-base"
+    ],
+    "globals": {
+        "Atomics": "readonly",
```
Comment (hamilton) [inline .eslintrc.json]: out of curiosity, is there a reason for these global rules?

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true
+    },
+    "extends": [
+        "airbnb-base"
```
Comment (hamilton) [inline .eslintrc.json:7]: Fantastic – I believe we essentially use this on Iodide, so this is welcome (for me)

File: public/index.html [hunk 3]
```diff
@@ -4,12 +4,11 @@
 	<meta charset='utf8'>
 	<meta name='viewport' content='width=device-width'>
 
-	<title>Svelte app</title>
+	<title>Metrics Dashboard (v2)</title>
 
 	<link rel='icon' type='image/png' href='/favicon.png'>
 	<link rel='stylesheet' href='/global.css'>
 	<link rel='stylesheet' href='/bundle.css'>
-	<script src='https://cdn.plot.ly/plotly-latest.min.js'></script>
```
Comment (hamilton) [inline public/index.html:12]: thanks for removing this

**Review by hamilton — APPROVED** [review]
This is great – thanks @robhudson. Had a couple of minor questions + went ahead and also linted the rollup file as well. Feel free to merge at your convenience if the comments don't merit anything else

File: .eslintrc.json (hunk 1 above)
Comment (robhudson) [reply .eslintrc.json]: I used `eslint --init` and it added these. I'm not sure what they do but I'll review. Thanks for calling this out.

**Pull request merged**